.github/workflows/release.yml

```diff
@@ -102,22 +102,14 @@ jobs:
         run: dart run grinder protobuf pkg-pub-deploy
         env: {PUB_CREDENTIALS: "${{ secrets.PUB_CREDENTIALS }}"}
 
-  deploy_sass_api:
-    name: Deploy sass_api
-    runs-on: ubuntu-latest
-    needs: [deploy_pub]
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/util/initialize
-        with: {github-token: "${{ github.token }}"}
-
-      - name: Deploy
-        run: dart run grinder deploy-sass-api
-        env:
-          PUB_CREDENTIALS: "${{ secrets.PUB_CREDENTIALS }}"
-          GH_TOKEN: "${{ secrets.GH_TOKEN }}"
-          GH_USER: sassbot
+      - name: Get Sass API version
+        id: sass-api-version
+        run: |
+          echo "version=$(cat pkg/sass_api/pubspec.yaml | sed -nE 's/version: (.*)/\1/p')" | tee --append "$GITHUB_OUTPUT"
+      # This should be /-separated rather than hyphenated, but pub.dev doesn't
+      # currently allow that (dart-lang/pub-dev#8690).
+      - run: git tag sass-api-${{ steps.sass-api-version.outputs.version }}
+      - run: git push --tag
```

Comment:
If you want to push with the implicit `${{ github.token }}` that comes from github actions, you need to add permissions to the job definition:
```
permissions:
  content: write
```

Reply:
Also, another problem I remember is that pushing with implicit token won't trigger another workflow. So you probably should use `${{ secrets.GH_TOKEN }}` in `checkout` action, that it will trigger the workflow for new tag push.

Reply:
Isn't that inherited from the invocation in `ci.yml`? `deploy_sass_parser` is pushing a tag without explicit local permissions below.

Reply:
> Also, another problem I remember is that pushing with implicit token won't trigger another workflow. So you probably should use `${{ secrets.GH_TOKEN }}` in `checkout` action, that it will trigger the workflow for new tag push.

Done.

Reply:
> Isn't that inherited

Permission inheritance in reusable workflow is a bit complicated. The permission declared in parent workflow defines the maximum permission child workflows can ask for, but if a child workflow did not explicitly ask for the permission, it won't get inherit from parent automatically.

https://docs.github.com/en/actions/sharing-automations/reusing-workflows#access-and-permissions

Anyways, better to use a PAT so that it guarantee the push event triggers a new workflow.

Reply:
Well, it's now using the same setup as `deploy_sass_parser` including the token it uses for checkout so it _should_ work.